websub-ballerina/Package.md

```diff
@@ -25,228 +25,43 @@ verification
- be delivered with respect to a particular topic.
- 
-For example, assume a scenario in which you receive notifications either when an issue is opened or when an issue is closed by subscribing to a particular topic in an issue tracking system. With a custom subscriber service listener, which extends the 
-generic WebSub subscriber service listener, you can allow two resources to accept content delivery requests (e.g., `onIssueOpened` and `onIssueClosed`) instead of the `onNotification` resource.
-
-These resources will accept two parameters:
-1. The generic `websub:Notification` record as the first parameter
-2. A custom record corresponding to the expected (JSON) payload of the notification (e.g., `IssueCreatedEvent`,
-`IssueClosedEvent`)
-
-You can introduce a specific service as such by extending the generic subscriber service listener, specifying a 
-mapping between the expected notifications and the resources that requests need to be dispatched to.
-
-The mapping can be based on one of the following indicators of a notification request. 
-(Requests will then be dispatched based on the value of the indicator in the request and a pre-defined mapping.)
-
-- A request header 
-
-    Dispatching will be based on the value of the request header specified as `topicHeader`. 
-    
-    ```ballerina
-    websub:ExtensionConfig extensionConfig = {
-        topicIdentifier: websub:TOPIC_ID_HEADER,
-        topicHeader: "<HEADER_TO_CONSIDER>",
-        headerResourceMap: {
-            "issueOpened": ["onIssueOpened", IssueOpenedEvent],
-            "issueClosed": ["onIssueClosed", IssueClosedEvent]
-        }
-    };
-    ```
-    
-    The `"issueOpened": ["onIssueOpened", IssueOpenedEvent]` entry indicates that when the value of the
-    `<HEADER_TO_CONSIDER>` header is `issueOpened`, dispatching should happen to a resource named `onIssueOpened`. 
-    
-    The first parameter of this resource will be the generic `websub:Notification` record, and the second parameter will 
-    be a custom `IssueOpenedEvent` record mapping the JSON payload received when an issue is created.
-
-- The payload: the value of a particular key in the JSON payload
-
-    Dispatching will be based on the value in the request payload of one of the map keys specified in the 
-    `payloadKeyResourceMap` map.
-    
-    ```ballerina
-    websub:ExtensionConfig extensionConfig = {
-        topicIdentifier: websub:TOPIC_ID_PAYLOAD_KEY,
-        payloadKeyResourceMap: {
-            "<PAYLOAD_KEY_TO_CONSIDER>": {
-                "issueOpened": ["onIssueOpened", IssueOpenedEvent],
-                "issueClosed": ["onIssueClosed", IssueClosedEvent]
+    websub:ListenerConfiguration listenerConfigs = {
+        secureSocket: {
+            key: {
+                path: "<KEYSTORE_PATH>",
+                password: "<KEYSTORE_PASSWORD>"
```

Comment:
Let's change these to certs and keys as in https://github.com/ballerina-platform/ballerina-distribution/blob/master/examples/basic-https-listener-client/basic_https_listener.bal